packages/lodestar/test/unit/chain/stateCache/stateContextCheckpointsCache.test.ts

```diff
@@ -0,0 +1,28 @@
+import {config} from "@chainsafe/lodestar-config/default";
+import {BenchmarkRunner} from "@chainsafe/lodestar-utils/test_utils/benchmark";
+import {ssz} from "@chainsafe/lodestar-types";
+import {generateCachedState} from "../../../utils/state";
+import {CheckpointStateCache} from "../../../../src/chain/stateCache";
+
+describe("CheckpointStateCache perf tests", function () {
```

Comment:
You are running a performance test in the unit tests run. You must ensure that everything run in the `yarn test:unit` completes very quickly ideally in < 100 ms.

This perf test must be run through a different command if applicable.